### ISSUE TITLE
[Website] Maintain and use DOAP file, or delete it

### ISSUE THREAD

**Issue by asfimport** [apache/arrow#25271]
ARROW-295 added a DOAP file so that Arrow could be listed at https://projects.apache.org/. Unfortunately, the next step of registering it in https://svn.apache.org/repos/asf/comdev/projects.apache.org/trunk/data/projects.xml never happened, so we're still not in the list.

Moreover, this file is not maintained, so it still lists the current Arrow version as 0.12.0.

**Reporter**: [Neal Richardson](https://issues.apache.org/jira/browse/ARROW-9165) / @nealrichardson

<sub>**Note**: *This issue was originally created as [ARROW-9165](https://issues.apache.org/jira/browse/ARROW-9165). Please see the [migration documentation](https://github.com/apache/arrow/issues/14542) for further details.*</sub>

**Comment by alamb** [comment]
We got a note from comdev last month:

-----

Dear Arrow PMC,

TL;DR: You're missing a DOAP file. Details at https://projects.apache.org/create.html

As you may know, the Community Development committee (ComDev) maintains a website [projects.apache.org](http://projects.apache.org/) which lists all ASF projects, and some basic details about them. These details are derived from DOAP (Description Of A Project) file that is maintained by each PMC.

Your project does not have a DOAP file.

This file is also referenced elsewhere, when such data is needed. When it's missing, the information can sometimes be sourced from other places, but sometimes this will result in your project being missing from various lists. For example, the listing of projects by category, or by programming language (https://projects.apache.org/projects.html?language) does not list your project, and thus you will not be discoverable by people looking for a project to get involved in.

The instructions for creating and maintaining your DOAP file, along with a handy took for generating a file in the correct format based on a few questions.

If you need any assistance in creating this file, please drop by the comdev mailing list, https://lists.apache.org/list.html?dev@community.apache.org

--Rich, for Comdev

**Comment by kou** [comment]
Ah, it seems that we need to add `<location>https://arrow.apache.org/arrow.rdf</location>` to https://svn.apache.org/repos/asf/comdev/projects.apache.org/trunk/data/projects.xml manually.

**Comment by alamb** [comment]
Here is a proposed PR to add an appropriate rdf file (which we will then link): https://github.com/apache/arrow/pull/37865

**Comment by alamb** [comment]
I will monitor https://projects.apache.org/project.html?arrow to see if it picks up changes

Also I have a proposal for updates to the DOAP file: https://github.com/apache/arrow-site/pull/408

**Comment by kou** [comment]
Thanks!
I confirmed that our entry is added to https://svn.apache.org/repos/asf/comdev/projects.apache.org/trunk/data/projects.xml .
But https://projects.apache.org/project.html?arrow doesn't pick up it yet...

**Comment by alamb** [comment]
https://projects.apache.org/project.html?arrow

Now appears to show the updated DOAP contents:

<img width="1232" alt="Screenshot 2023-09-27 at 6 46 04 AM" src="https://github.com/apache/arrow/assets/490673/468de967-3056-4758-a1c4-473ab6269ecc">

**Comment by kou** [comment]
Yay!
